src/pages/PlayReason.tsx

```diff
@@ -84,6 +107,7 @@ const PlayReason: React.FC<RouteComponentProps> = (props) => {
               onChange={(e) => {
                 setState(e.target.value);
                 setError("");
+                console.log(e);
```

Comment:
can we remove this?

Reply:
whoops, seems I left that in there by accident, it's fixed now @lucycy98